lib/tether/tether.go

```diff
@@ -251,11 +251,25 @@ func (t *tether) setHostname() error {
 }
 
 func (t *tether) setNetworks() error {
-	for _, v := range t.config.Networks {
-		if err := t.ops.Apply(v); err != nil {
+
+	// internal networks must be applied first
+	for _, network := range t.config.Networks {
+		if !network.Internal {
+			continue
+		}
+		if err := t.ops.Apply(network); err != nil {
```

Comment:
Being pedantic here - this block can be made terse with

```go
if network.Internal && t.ops.Apply(network) != nil {
    ...
}
```

This is less readable than the current form though, so we can leave it as-is.